src/libraries/System.Net.Http/tests/FunctionalTests/AssemblyInfo.cs

```diff
@@ -3,5 +3,7 @@
 // See the LICENSE file in the project root for more information.
 
 using Xunit;
+using Microsoft.DotNet.XUnitExtensions.Attributes;
 
 [assembly: SkipOnCoreClr("System.Net.Tests are flaky and/or long running: https://github.com/dotnet/runtime/issues/131")]
+[assembly: SkipOnMono("System.Net.Tests are flaky and/or long running: https://github.com/dotnet/runtime/issues/131")]
```

Comment:
These attributes are only about stress modes, right? What stress are we running on mono in ci?

(I still think the attributes need to either be renamed or changed to mean what they say.)

cc: @safern

Reply:
SkipOnMono is about disabling on Mono at all because the runtime is not yet complete from my understanding. 

But yeah for coreclr I was planning on changing it this week when I get some free time for that.

Reply:
If I read the code in https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.XUnitExtensions/src/Discoverers/SkipOnCoreClrDiscoverer.cs correctly the way it's used here would skip on CoreCLR all the time.

There's an overload that takes a stress mode (https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.XUnitExtensions/src/Attributes/SkipOnCoreClrAttribute.cs) but that's not used here.

Reply:
I agree that we should replace all usages of SkipOn* without the overload for special situations with `ActiveIssue` instead.

Reply:
> correctly the way it's used here would skip on CoreCLR all the time.

Not really. It skips it only if running in a checked CoreLib.

https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.XUnitExtensions/src/Discoverers/SkipOnCoreClrDiscoverer.cs#L34

`ActiveIssue` will skip for all cases, which we want to avoid as these tests are only long running in Checked CoreCLR and on Mono.

I will update SkipOnCoreCLR to be more clear so that even Checked has to be a flag passed down as a parameter when we want to skip it always when running on a checked CoreLib.

For SkipOnMono we don’t even have stress modes as arguments because I was under the impression we don’t do stress runs on mono runtime for libraries tests and we wanted to use this to skip the test always when running in a mono runtime, which is the equivalent of the current RSP files.

Reply:
> I will update SkipOnCoreCLR to be more clear so that even Checked has to be a flag passed down as a parameter when we want to skip it always when running on a checked CoreLib.

Thanks, @safern; I appreciate it.  So once you make that change, SkipOnCoreCLR without any flags will mean "always skip this when running on any variant of coreclr", right?  So presumably you'll also modify all SkipOnCoreCLR usage then to pass flags, unless we actually want it skipped everywhere?

Reply:
> So once you make that change, SkipOnCoreCLR without any flags will mean "always skip this when running on any variant of coreclr", right?

Right :smile:

>  So presumably you'll also modify all SkipOnCoreCLR usage then to pass flags, unless we actually want it skipped everywhere?

Correct I will do the needed changes in runtime repo to react to that behavior change.

Reply:
Ah ok, in that case something else is going on as Mono shouldn't show the significant slowdown that Checked CoreCLR has. Weird.

Reply:
Opened: https://github.com/dotnet/arcade/pull/4724 for the desired behavior of `SkipOnCoreClr`